blog/docs/articles/aws_to_func_migration.md

```diff
@@ -0,0 +1,103 @@
+# Migrating Functions from AWS Lambda to Knative Functions using Golang
+
+**Author: Matthias Weßendorf, Senior Principal Software Engineer @ Red Hat**
+
+_In a [previous post](/blog/articles/consuming_s3_data_with_knative){:target="_blank"} we discussed the consumption of notifications from an AWS S3 bucket inside a Knative Function. This post will describe the migration from a AWS Lambda Function, receiving S3 notifications, to Knative Functions._
+
+With Serverless Functions one of the common use-cases is to execute custom code based on an event trigger, like a notification from the AWS S3 service. With AWS Lambda you can run those programs on Amazon's cloud offerings, but running the code on your own data-center is much harder.
```

Comment:
There should be a reference to Knative Functions docs, either install section or "how to start".

Reply:
Sort of prerequisite intro section.

Reply:
The referenced article has a little intro on func already.

But I can add some like link to "getting started"

Reply:
The current version with note to Functions docs is fine. Thanks!

I'll resolved this thread.